src/Type/Definition/IDType.php

```diff
@@ -66,14 +64,11 @@ public function parseValue($value)
     }
 
     /**
-     * @param Node         $valueNode
      * @param mixed[]|null $variables
      *
-     * @return string|null
```

Comment:
can move to return type

Reply:
Unfortunately, not; that violates the contract with `LeadScalar`. But I was able to typehint the return of `parseValue`.